### PR TITLE
Bar chart: Fix stacking bug when data produces 0 accumulators

### LINKED ISSUE #49077: Bar chart displays no bars when all values in one Row are 0 and stacking is 100%
**What happened**:
All bars disappeared in the Bar chart

**What you expected to happen**:
Only the chart which contains the zeros should disappear

**How to reproduce it (as minimally and precisely as possible)**:
1. Create a new Bar chart panel
2. Set Stacking -> 100%
3. Use testdata DB to input this example csv:
```csv
name, number, number2, number3, number4
Name1, 40, 5, 20, 10
Name2, 0, 0, 0, 0
Name3, 6, 3, 5, 1
Name4, 1, 1, 1, 2
```
4. No bar is displayed:
image

6. Change a zero to any other number:
```csv
name, number, number2, number3, number4
Name1, 40, 5, 20, 10
Name2, 0, 0, 0, 1
Name3, 6, 3, 5, 1
Name4, 1, 1, 1, 2
```
7. Bars are displayed correctly

**Environment**:
- Grafana version: 8.5.2 (90701be19)
- Data source type & version:  Loki v2.5.0 but also TestData DB
- OS Grafana is installed on: Windows
- User OS & Browser: Windows. Firefox and Chrome
- Grafana plugins: -
- Others: This happens on https://play.grafana.org/ as well

### PR DESCRIPTION
**Pull request by dprokop**
fixes #49077  ... 

But it's not so elegant - any thoughts @leeoniya ?

### CODE REVIEW

File: public/app/plugins/panel/barchart/bars.ts [hunk 1]
```diff
@@ -307,11 +307,16 @@ export function getConfig(opts: BarsOptions, theme: GrafanaTheme2) {
...
+          divider = alignedTotals![seriesIdx][dataIdx]!;
+        }
+
+        const v = divider === 0 ? 0 : raw / divider;
         // Format Values and calculate label offsets
-        const text = formatValue(
-          seriesIdx,
-          rawValue(seriesIdx, dataIdx)! / (pctStacked ? alignedTotals![seriesIdx][dataIdx]! : 1)
-        );
+        const text = formatValue(seriesIdx, v);
```
Comment (leeoniya) [inline public/app/plugins/panel/barchart/bars.ts:319]: are all the changes in this block equivalent to just adding `|| 1` to the pctStacked case? :thinking: 

```js
rawValue(seriesIdx, dataIdx)! / (pctStacked ? (alignedTotals![seriesIdx][dataIdx]! || 1) : 1)
```

**Comment by leeoniya** [comment]
> But it's not so elegant

nothing much better comes to mind :shrug: 

would be good to add this case to the stacking gdev dashboard

**Comment by dprokop** [comment]
> would be good to add this case to the stacking gdev dashboard

Added @leeoniya

**Review by leeoniya — APPROVED** [review]
thanks, lgtm :shipit:

**Pull request merged**